ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -828,16 +828,13 @@ public void setKeyboardType(ReactEditText view, @Nullable String keyboardType) {
       // This will supercede secureTextEntry={false}. If it doesn't, due to the way
       //  the flags work out, the underlying field will end up a URI-type field.
       flagsToSet = InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD;
-<<<<<<< HEAD
     } else if ((view.getStagedInputType() & AUTOCAPITALIZE_FLAGS) != 0) {
       // This prevents KEYBOARD_TYPE_FLAGS from being unset when the keyboardType is
       // default, null, or unsupported, and autocapitalize is on.
       // Unsetting these flags breaks the autoCapitalize functionality.
       unsettingFlagsBreaksAutocomplete = true;
```

Comment:
I think you need to remove this section to match how things have been done in 0.66:
<img width="666" alt="Screenshot 2022-02-01 at 16 54 29" src="https://user-images.githubusercontent.com/16104054/152017146-eee6c7c0-4266-4290-be57-07a75c72e6c2.png">

Reply:
```suggestion
```